library/cohesity_clone_vm.py

```diff
@@ -0,0 +1,392 @@
+
+# !/usr/bin/python
+# Copyright (c) 2018 Cohesity Inc
```

Comment:
2019 ? 

Also make this consistent and updated for 2020

Reply:
Ok will change that